src/e2e/resources/data/InstructorNotificationsPageE2ETest.json

```diff
@@ -0,0 +1,103 @@
+{
+  "accounts": {
+    "INotifications.instr": {
+      "googleId": "tm.e2e.INotifications.instr",
```

Comment:
`Notifications` is a bit too long. Let's add a naming rule to shorten this to `Notifs`.